project1/client/client_main.cpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * @brief Entry point for the FTP client.
+ */
+
+
+#include "client.h"
+#include "input_parser/input_parser.h"
+#define PORT 8080
+namespace client {
+
+
+    int main(int argc, const char **argv) {
+        Client c;
```

Comment:
(minor) Good practice is to check the number of arguments and print a help message if it's wrong. Check out what I did in the equivalent server code.